aten/src/ATen/core/type.cpp

```diff
@@ -389,6 +389,15 @@ bool Type::isSubtypeOfExt(const TypePtr rhs, std::ostream* why_not) const {
   return false;
 }
 
+bool Type::is_module() const {
+  if (auto cls_type = this->cast<ClassType>()) {
```

Comment:
`ClassType` and `InterfaceType` should override `is_module()`, let's not do a cast here.

Reply:
oops yeah they do overwrite it, this cast is not needed, I will remove.